data/containers/autoyast_containers.xml.ep

```diff
@@ -119,7 +179,7 @@
       % }}
```

Comment:
Seems like one is redundant over here

Reply:
@m-dati so you are saying this is needed, otherwise the CI fails, right?